testmodule/README.md

```diff
@@ -0,0 +1,16 @@
+Example modules for test of Mold
+
+# How to add a new one (Linux/Mac binary)
+## Rust
+1. Write a code in `test-module/tests/`
+2. Update ``[[bin]]`` in `test-module/Cargo.toml`
+
+# How to add a new one (Python)
+Not supported yet
+
+# How to add a new one (TypeScript)
+Not supported yet
+
+# How to add a new one (WASM Binary)
+Not supported yet
+
```

Comment:
README should start with the first level header.
Please only one first level header in the README.md file.

Reply:
Ok thanks